pyro/distributions/util.py

```diff
@@ -1,24 +1,31 @@
 from __future__ import absolute_import, division, print_function
 
-import functools
-import os
-
 import torch
 import torch.nn.functional as F
 from torch.autograd import Variable
 
-# TODO Decide based on torch.__version__ once torch.distributions matures.
-USE_TORCH_DISTRIBUTIONS = int(os.environ.get('PYRO_USE_TORCH_DISTRIBUTIONS', 0))
-
 
-def torch_wrapper(pyro_dist):
+def copy_docs_from(source_class):
```

Comment:
heh this is cute. are we eventually going to migrate them completely so theres only one distribution? will it be `pyro.distributions` or `pyro.distributions.torch`?

Reply:
We're planning to eventually migrate docs. Thereafter we can still use this decorator to copy docs from `Distributions` to instance classes.